TESTOrderScript.js

```diff
@@ -1,17 +1,24 @@
+// TODO: personal pref and completely optional: my last 2 teams have alphabetized their .js/.ts files, so that when everything is collapsed it's a bit easier to find stuff.
+//  it's pretty nit-picky so don't worry about it if you don't want, but it seems like something you'd like to have :)
+
 //addRow Objective: find the 'entry' number of the last/newest created order fields row, to then add 1 to it and make that the number on the next row that will be created
 //runs when the Add more button is clicked
 function addRow() {
     //find the element with 'newest' class on it and look through its classes for the one that contains 'entry'
+    // TODO: rather than split and loop through the classNames, maybe you could just use .find or .filter to get the current class that way?
     const classList = document.getElementsByClassName('newestRow')[0].className.split(/\s+/);
     classList.forEach(currentClass => {
         if ((currentClass.includes)('entry')) {
-            //convert the string to a number and remove the first five characters from the string, set a variable to whatever is left plus 1 
+            // TODO: I think there's a safer way to get that last character, try currentClass.slice(-1). The thought here is that, if you ever change the word 'entry', suddenly you have to remember to change the substring length, but by getting the last char, it doesn't matter what you call that class now
```

Comment:
I thought about this initially and I guessing the likelihood of a person ordering with more than 9 rows is higher than the likelihood of the word 'entry' changing. I could redo it with a character and then a split off that character. However, other parts of the function that this statement is in also use the word 'entry'. Thinking on this one.